jump-core/src/main/java/com/bitdecay/jump/collision/CollisionUtilities.java

```diff
@@ -54,26 +56,26 @@ private static boolean canSkipAxis(BitPoint axis, Manifold manifold) {
      *
      * <b>NOTE:</b> Due to logic to make sure the feel is correct, there are situations (such as one-way
      * platforms) that can result in NO axis (aka the Zero-axis) being set on the resolution.
-     *
-     *  @param body the body being resolved
+     * @param body the body being resolved
      * @param otherBody the other body that participated in the collision
-     * @param cumulativeResolution the current partially built resolved position
      */
-    public static Manifold solve(ManifoldBundle bundle, BitBody body, BitBody otherBody, BitPoint cumulativeResolution) {
+    public static Manifold solve(ManifoldBundle bundle, BitBody body, BitBody otherBody) {
         bundle.getCandidates().sort((o1, o2) -> Float.compare(Math.abs(o1.distance), Math.abs(o2.distance)));
 
-        BitPoint relativeMovement = CollisionUtilities.getRelativeMovement(body, otherBody, cumulativeResolution);
+        BitPoint relativeMovement = CollisionUtilities.getRelativeMovement(body, otherBody);
 
         float dotProd;
         for (Manifold manifold : bundle.getCandidates()) {
             dotProd = relativeMovement.dot(manifold.axis);
             if (dotProd != 0 && !MathUtils.sameSign(dotProd, manifold.distance)) {
                 if (otherBody instanceof TileBody) {
                     // confirm that body came from past this thing
-                    float resolutionPosition = body.aabb.xy.plus(cumulativeResolution).plus(manifold.result).dot(manifold.axis);
-                    float lastPosition = body.lastPosition.dot(manifold.axis);
 
-                    if (CollisionUtilities.canTileCollisionCanBeSkipped((TileBody) otherBody, manifold, resolutionPosition, lastPosition)) {
+                    // use the relative position between the centers of the bodies
```

Comment:
Might be a meaningless change. This way makes more sense to me, but it also has more math (therefore slower). Might make sense to just change it back if it wasn't broken.